package.json

```diff
@@ -91,5 +91,8 @@
   },
   "engines": {
     "node": ">=18"
+  },
+  "prettier": {
```

Comment:
What is this?

Reply:
Prettier can enforce its rules using a `prettier` key in `package.json` or with a separate file. The default option for Prettier is to always wrap single parameters in arrow functions with parentheses. This meant that when formatting code with Prettier,
```js
test('test name', t => { ... });
```
would become
```js
test('test name', (t) => { ... });
```
This option disables that formatting.

Reply:
ShellJS isn't configured to use `prettier` and we don't have that package as a dependency. Realistically, I can't accept configs for stuff that I don't use for this project.

I won't object if you want to use `prettier` for your own changes, as long as the final PR also passes `npm run lint`. But you would have to find a way to do this without adding extra configs in our project.

Reply:
That's understandable, I've removed it.